engine/include/cubos/engine/renderer/viewport.hpp

```diff
@@ -0,0 +1,26 @@
+/// @file
+/// @brief Component @ref cubos::engine::Viewport.
+/// @ingroup renderer-plugin
+
+#pragma once
+
+#include <glm/vec2.hpp>
+
+#include <cubos/core/reflection/external/glm.hpp>
```

Comment:
This header is only needed on the `.cpp`, which is were you actually define the reflection. Here, `reflect.hpp` is enough to get the `CUBOS_REFLECT`.